cvmfs/shrinkwrap/shrinkwrap.cc

```diff
@@ -162,14 +164,44 @@ int main(int argc, char **argv) {
         params.spec_trace_path = optarg;
         break;
       case 'j':
-        params.num_parallel = atoi(optarg);
-        break;
-      case 'n':
-        params.retries = atoi(optarg);
+        errno = 0;
+        endptr = NULL;
+        tmp_value = strtoll(optarg, &endptr, 10);
```

Comment:
This now is nearly to `String2Uint64Parse` and checks that the option was fully and correctly parsed. I can refactor to use the existing function if you like, but I did not directly as I would have needed to convert optarg to String. This would be a small change so please let me know. @jblomer

Reply:
@nhazekam yes, please go ahead.